circle.yml

```diff
@@ -73,6 +73,6 @@ deployment:
       - curl -s --fail $DOCS_URL | bash -s -- -r docs/requirements.txt $CIRCLE_BRANCH
   bintray:
     tag: /[0-9]+(\.[0-9]+){2}(-alpha|-beta|-rc[0-9]+)?(\+[0-9]{3})?/
-    owner: palantir
+    owner: atlasdb
```

Comment:
Are you sure this is enough? I think the internal build used to have some difficulty publishing externally as it wasnt allowed by the firewall.

Reply:
I've asked some people internally, and it seems we should be able to publish to bintray from internal circle. I've also moved the required auth tokens internally. I'll see how it goes later this week, when we try to publish, but now I think this should be enough.